packages/components/.storybook/preview.tsx

```diff
@@ -26,4 +30,26 @@ export const parameters = {
       },
     ],
   },
+  badgesConfig: {
+    deprecated: {
+      ...defaultBadgesConfig[BADGE.DEPRECATED],
+      tooltip: "This component is deprecated, please avoid using it.",
+    },
+    beta: {
+      ...defaultBadgesConfig[BADGE.BETA],
+      tooltip: {
```

Comment:
nit: could we remove the tooltips for now, until we find a use for them? 🙂

Reply:
I'm down for removing the tooltips, mostly to show configuration possibility